frontend/pages/policies/ManagePoliciesPage/_styles.scss

```diff
@@ -24,34 +24,6 @@
       .dropdown__help-text {
         color: $ui-fleet-black-50;
       }
-      .is-disabled * {
```

Comment:
Looks like these changes knocked the style of these disabled options off a bit, we'll want to make sure that stays consistent

Reply:
I realized the color was 50 not 25, updated that. Let me know if you notice something else that is incorrectly styled.

Reply:
Not sure if you are saying you updated the color from `main` intentionally or if you restored this branch to what was on `main`. If the former, I trust that whatever you are referencing for that color is correct. If the latter, it's not the same so please fix.

Also, italicization is broken
![Screenshot 2024-04-11 at 12 12 20 PM](https://github.com/fleetdm/fleet/assets/61553566/eb84c4dd-ca52-4558-9375-4a0de048392a)
: